crowdsourcing/serializers/project.py

```diff
@@ -36,12 +36,17 @@ class ModuleSerializer(DynamicFieldsModelSerializer):
     deleted = serializers.BooleanField(read_only=True)
     template = TemplateSerializer(many=True, read_only=False)
     module_tasks = TaskSerializer(many=True, read_only=True)
+    csv_data = serializers.ListField(write_only=True)
+    has_data_set = serializers.BooleanField(write_only=True)
```

Comment:
How can it be both read_only and write_only? We only want it when creating/updating the instance hence write_only?